che-incubator/che-code
# [WIP] Disable terminal auto-opening

Disable terminal auto-opening until https://github.com/eclipse/che/issues/21537 is fixed

#### How to test
1. Spin up a DevWorkspace using the following link:
che-host#https://github.com/azatsarynnyy/java-spring-petclinic/tree/devfilev2
2. The terminal should not be opened automatically on Che-Code start. It should be possible to open the terminal with the `Terminal > New Terminal` command or with the <kbd> Ctrl+`</kbd> hotkey.
